### ISSUE TITLE
🎈 Off-topic | What's your theme name 😅

### ISSUE THREAD

**Issue by yedhrab** [Denis-net/vscode-ahk-manager#19]

**Comment by Denis-net** [comment]
Hello @yedhrab 😊👋🏻


TL;DR; ["Dobri Next -A00- Black"](https://marketplace.visualstudio.com/items?itemName=sldobri.bunker)

Long version:
In Sublime I was used to set the "Material theme" and 
When I started with VSC I was really looking for something more darker, that don't hurt my eyes, that simplify my mental chaos and after a long search I found this theme, more beautiful (in my opinion) that the Material ones (Which i still like) :
[Dobri Next](https://marketplace.visualstudio.com/items?itemName=sldobri.bunker), which has a wonderful collection too.

But a new problem arose: what should i choose ?
I have just one VSC instance and i don't want to miss any of them.
And it was then that I found it, something that opened a new world to me, a simple extension:

[Random Theme Switcher](https://marketplace.visualstudio.com/items?itemName=zanza00.random-theme-switcher) made by a great developer.

> It opened me to the VSC extensions writers world.
So, i made a contribution to that extension .
I highly recommend you to give it a try, you'll never regret it.
Another rich pack of themes that i found interesting is [this one](https://marketplace.visualstudio.com/items?itemName=daylerees.rainglow).
There are a lot of great theme out there, don't stop here 😉

Another thing that i suggest you to do is: play with the settings, 
because you can reduce the scroll bars width, for example, or change colors that you don't like (the bottom bar, cough cough...)

I hope this could help you to satisfy or increase your curiosity.
May you have a good evening !

P.S. a Theme that i found very good and i would recommend you to try is [this one](https://github.com/yedhrab/DarkCode-Theme) 😉

**Comment by yedhrab** [comment]
Thank you so much, I looked all of them 😊

> [The last theme](https://github.com/yedhrab/DarkCode-Theme) that you suggest is mine 😇

**Comment by Denis-net** [comment]
Yes: The last but not the least 😉✌🏻
Thank you too 😄👍🏻

**Comment by rediffusion** [comment]
**Hello guys**, interested in discussing. Want to share my themes:

- Dracula Official > Dracula Soft
- Ayu Adaptive > Ayu One
- Flatland   Monokai Theme
- Noctis > Minimus, Hibernus, Lilac
- Sublime Material Theme > Dark
- eppz! (C# theme for Unity) ••• the best for in my opinion!
- Flatland Monokai Theme
- One Dark for Sublime Babel

 _Cheers_! 👍✌️
